### ISSUE TITLE
[ext/web] `Event.timeStamp` is always 0

### ISSUE THREAD

**Issue by lowlighter** [denoland/deno#23467]
Version: Deno 1.42.4

Currently any `Event.timeStamp` is always set to 0
https://github.com/denoland/deno/blob/472a37064071c66cd1311cdea2e78de8d2bc0641/ext/web/02_event.js#L156

> The `timeStamp` read-only property of the [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event) interface returns the time (in milliseconds) at which the event was created.
>
> https://developer.mozilla.org/en-US/docs/Web/API/Event/timeStamp

It should be set to `performance.now()` or similar to match more closely what is expected

**Motivation**
Make possible to test cases that requires this property to be non-zero:
```ts
const oldEvent = new Event()
await delay(5)
const lastEvent = new Event()
a.lastUpdated = oldEvent.timeStamp+1

a.addEventListener(event => {
  if (event.timeStamp < a.lastUpdated)
    return
  expect(event).toBe(lastEvent)
})

a.dispatchEvent(oldEvent)
a.dispatchEvent(lastEvent)
```
The above is not possible currently, because `timeStamp` property is readonly

**Comment by dsherret** [comment]
This is a documented spec deviation: https://docs.deno.com/runtime/manual/runtime/web_platform_apis#spec-deviations-1

**Comment by lowlighter** [comment]
Is there a specific motivation behind this spec deviation ?

**Comment by dsherret** [comment]
Performance: https://github.com/denoland/deno/pull/20191

**Comment by lowlighter** [comment]
Yeah it'd be nice, or at least somehow expose the `_attributes` symbol so users are able to override properties from the event despite them being readonly

**Comment by lucacasonato** [comment]
Mh, I still wonder if we can re-implement this in some way...
